### PR TITLE
feat: show BIC balance in account details

### LINKED ISSUE #8145: [Task]: show BIC balance in account details
### Task description

- Add the BIC of each account
- Hide balance of each explicit account

### PR DESCRIPTION
**Pull request by cpl121**
Closes #8145 

To try it for the first time, you must update `syncBalance` for the first time in one of the following ways
- Adding `syncBalance(initialSelectedWalletId, true)` in the login in one of the final steps
- Creating an account or making a transfer (launch `handleNewOutputEvent` or `handleSpentOutputEvent`)

## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/lib/core/wallet/interfaces/extended-balance.interface.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { Balance } from '@iota/sdk/out/types'
 
 export interface IBalance extends Balance {
-    blockIssuanceCredits?: number
+    blockIssuanceCredits?: { [accountId: string]: number }
+    totalWalletBic?: number
```
Comment (marc2332) [inline packages/shared/lib/core/wallet/interfaces/extended-balance.interface.ts:5]:
```suggestion
    blockIssuanceCredits: { [accountId: string]: number }
    totalWalletBic: number
```

File: packages/shared/lib/core/wallet/actions/syncBalance.ts [hunk 2]
```diff
@@ -8,10 +8,16 @@ import { IBalance } from '../interfaces'
...
+    const blockIssuanceCredits = syncCongestion ? await getBicWalletBalance(walletId) : {}
+    const totalWalletBic =
+        syncCongestion && blockIssuanceCredits.length > 0
+            ? Object.values(blockIssuanceCredits).reduce((acc, bic) => acc + Number(bic), 0)
+            : 0
+
     const totalBalance: IBalance = {
         ...balances,
         ...(syncCongestion && !!blockIssuanceCredits && { blockIssuanceCredits }),
+        ...(syncCongestion && !!totalWalletBic && { totalWalletBic }),
```
Comment (marc2332) [inline packages/shared/lib/core/wallet/actions/syncBalance.ts:20]:
```suggestion
        ... { totalWalletBic, blockIssuanceCredits }
```

File: packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/FundConfirmationView.svelte [hunk 3]
```diff
@@ -42,7 +42,7 @@
     function getTotalAvailableMana(): number {
         return (
             getManaBalance($selectedWallet?.balances?.mana?.available) +
-            ($selectedWallet?.balances.blockIssuanceCredits ?? 0) -
+            ($selectedWallet?.balances.totalWalletBic ?? 0) -
```
Comment (marc2332) [inline packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/FundConfirmationView.svelte:45]:
```suggestion
            ($selectedWallet?.balances.totalWalletBic) -
```

**Review by marc2332 — CHANGES_REQUESTED** [review]
wouldn't it better to keep things this simple

File: packages/shared/locales/en.json [hunk 4]
```diff
@@ -701,6 +701,7 @@
             },
             "details": {
                 "balance": "Balance",
+                "blockIssuanceCredits": "Block Issuance Credits Mana",
```
Comment (evavirseda) [inline packages/shared/locales/en.json:704]:
```suggestion
                "blockIssuanceCredits": "Block Issuance Credit (BIC)",
```

**Review by evavirseda — CHANGES_REQUESTED** [review]
Nice job, just a little detail:pray:

**Review by evavirseda — APPROVED** [review]
good job :rocket:

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
add logic to get BIC if there are explicit accounts to `buildWalletState.ts`

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**